roles/backup/tasks/main.yml

```diff
@@ -1,16 +1,28 @@
 ---
 
-- name: set up weekly backup
+- name: install backup script
   become: yes
   template:
-    src: etc/cron.weekly/cnx-backup
-    dest: /etc/cron.weekly/cnx-backup
+    src: usr/local/bin/cnx-backup
+    dest: /usr/local/bin/cnx-backup
     mode: 0755
 
-- name: set up slim dumps
+- name: set up weekly backup
+  become: yes
+  src: /usr/local/bin/cnx-backup
+  dest: /etc/cron.weekly/cnx-backup
+  state: link
```

Comment:
I tried running on a machine that already has the file at this location and saw this:

```
TASK [backup : set up weekly backup] *****************************************************************************************************************************************
fatal: [xenial-openstax-test3.local]: FAILED! => {"changed": false, "gid": 0, "group": "root", "mode": "0755", "msg": "refusing to convert between file and link for /etc/cron.weekly/cnx-backup", "owner": "root", "path": "/etc/cron.weekly/cnx-backup", "size": 580, "state": "file", "uid": 0}
```

Adding `force: yes` after this line fixes it.